tunder/test/console/contexts/sky_command_context.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:mason_logger/mason_logger.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:tunder/console.dart';
+import 'package:tunder/database.dart';
+import 'package:tunder/src/console/commands/migrations/contracts/migration_command.dart';
+
+class SkyCommandContext {
+  late final SkyCommand sky;
+  late final MigrationCommand command;
+  final logger = LoggerMock();
+  final progress = ProgressMock();
+  late final migrationsDir;
+
+  SkyCommandContext({required MigrationCommand forCommand}) {
+    command = forCommand;
+    sky = SkyCommand<int>(logger, silent: true)..addTunderCommand(command);
```

Comment:
The class properties don't need to be marked as late if they're initialized in the initializer list. Example:
```dart
SkyCommandContext({required MigrationCommand forCommand})
  : command = forCommand,
    sky = SkyCommand<int>(logger, silent: true)..addTunderCommand(command);
```

Reply:
It doesn't work though:
<img width="1195" alt="image" src="https://user-images.githubusercontent.com/3236791/183667930-a1cf758b-495e-4893-b09c-2c441f2aeca1.png">

Reply:
Oh, that's why you made it like so, got it.